tests/0081-admin.c

```diff
@@ -2705,11 +2707,209 @@ static void do_test_ListConsumerGroups(const char *what,
                 TEST_CALL_ERR__(rd_kafka_AdminOptions_set_request_timeout(
                     options, request_timeout, errstr, sizeof(errstr)));
         }
-
-
         topic             = rd_strdup(test_mk_topic_name(__FUNCTION__, 1));
         exp_mdtopic.topic = topic;
 
+        if (test_consumer_group_protocol_consumer()) {
```

Comment:
Yes, these are true but since the test is similar for other clients which do not have has_match_types functionality like in librdkafka, so only these were agreed upon. Otherwise, we would diverge a lot !